packages/amplication-client/src/Application/code-view/CodeViewExplorerTree.tsx

```diff
@@ -0,0 +1,120 @@
+import { Icon, TreeView } from "@amplication/design-system";
+import React, { useCallback, useEffect, useState } from "react";
+import { Build } from "../../models";
+import { FileDetails } from "./CodeViewPage";
+import { FileExplorerNode } from "./FileExplorerNode";
+import { NodeTypeEnum } from "./NodeTypeEnum";
+import { StorageBaseAxios, StorageResponseType } from "./StorageBaseAxios";
+import { useQuery } from "react-query";
+import { AxiosError } from "axios";
+import { remove } from "lodash";
+import { FileMeta } from "./CodeViewExplorer";
+import "./CodeViewBar.scss";
+
+const CLASS_NAME = "code-view-bar";
+
+type Props = {
+  selectedBuild: Build;
+  onFileSelected: (selectedFile: FileDetails) => void;
+};
+
+const NO_FILES_MESSAGE = "There are no available files to show for this build";
+
+const INITIAL_ROOT_NODE: FileMeta = {
+  type: NodeTypeEnum.Folder,
+  name: "root",
+  path: "/",
+  children: [],
+  expanded: false,
+};
+
+const CodeViewExplorerTree = ({ selectedBuild, onFileSelected }: Props) => {
```

Comment:
React component should be assign her type: const CodeViewExplorerTree: React.FC<Props>